lib/src/textArea/textArea.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="form-group text-area" [ngClass]="className">
```

Comment:
why so many wrapper

Reply:
this is a normal bootstrap layout

Reply:
![image](https://user-images.githubusercontent.com/49055952/80932700-3260de80-8df3-11ea-8469-f3b25e757dfc.png)

https://sebgroup.github.io/bootstrap/forms

This is a normal bootstrap layout ☝️ @DanSallau 
Ours has one extra wrapper. I simply want us to have a look at it and think about if we really need it. If we do that's fine.